doc/concept/unique-proofs-using-tries-merkleproofs.md

```diff
@@ -0,0 +1,109 @@
+Having an eventually consistent solution that publishes the internal changes in the registry, gives other parties in the network the ability to validate the soundness of the registry itself, and the end-users the ability proof the soundness of their claims using the same trie root.
+Publishing all the data involved in the transaction would give legal impediments to ETT group and expose sensitive information about the users - cryptographic hashing of the data mitigates this issue.
+
+## Sketch (optional)
+```mermaid
+graph LR
+    A[fa:fa-file-code A production certificate ] -->|claim x Whrs| C(Go shopping)
+    B[fa:fa-file-code B consumption certificate] --> C[Registry Processes transaction]
+    C -->|update local| D[fa:fa-database Database]
+    C -->|add subproof to | E[fa:fa-sitemap Merkle tree ]
+```
+
+## Solutions
+The merkle tree datastructure is promising in several ways and is in production for validation on blockchains such as ethereum that utilizes [Patricia Merkle Trees](https://ethereum.org/en/developers/docs/data-structures-and-encoding/patricia-merkle-trie/). The ethereum solution is in production and is used to validate transactions in a block to verify whether the transaction is included in the current block, by having shallower cryptographic proofs and subproofs there is no immediate need of going through the full block of transactions in the block.  
+
+This means that Merkle trees and their derivations has the utlity that they can batch transactions and by using the root can verify inclusion or not in the tree itself. This feature gives us the ability to batch large amount of transactions into a singled hashed value that can be published to validate the individual transactions and that the underlying data has not been changed in any way after its publication.
+
+The registry will thus consist of two loosely coupled mechanisms, a conventional database and a trie for validation of the transacations on the database. A batch of transactions can thus be appended the trie - for the proof of concept trie will consist of 2^20 entries approximately a million entries, will be available per batch for a registry before it will be published. There could be a timer that closes the batch if the transaction volume makes finalization an issue, the trie size could also be a variable determined by the number of users on the registry. 
+
+After a batch is processed by the registry - be it by time or by reaching the limit of the tree storage space the hash of the root of the trie can then be published and used for validating subproofs. The end-user can hence lookup the root in a log, at a blockchain etc and validate their energy consumption using the trie root as input to a subproof validation mechanism.
+
+A proof has 3 ways of being validating using this structure:
```

Comment:
```suggestion
A proof has 3 ways of being validated using this structure:
```

Reply:
What does "this structure" refer to? The merkle tree/trie structure?

Reply:
It is the trie/merkleproof structure indeed